docs/mmds/mmds-user-guide.md

```diff
@@ -276,6 +330,12 @@ The request was successfully processed and a response was successfully formed.
 
 The request was malformed.
 
+*401* - `Unauthorized`
```

Comment:
Should we also document the error case of mmdsv1 put?

Reply:
Done